src/js/modules/hasFontLoaded/test/index.test.js

```diff
@@ -0,0 +1,74 @@
+import FontFaceObserver from 'fontfaceobserver';
+import TestUtils from '@justeat/js-test-buddy';
+import { hasFontLoaded } from '../index';
+
+jest.mock('fontfaceobserver');
+
+describe('module hasFontLoaded', () => {
+    describe('when the fonts load correctly', () => {
+        beforeAll(() => {
+            FontFaceObserver.mockImplementation(() => ({
+                load: () => new Promise(resolve => {
+                    resolve();
+                })
+            }));
+        });
+
+        it('adds `webfonts-loaded` to the body when JustEatBasis webfont has loaded', () => {
+            // Arrange
+            TestUtils.setBodyHtml(`
+                <link href="https://d30v2pzvrfyzpo.cloudfront.net/fonts/JustEatBasis-Regular-optimised.woff2" type="font/woff2">
+                <body></body>
+            `);
+
+            // Act
+            hasFontLoaded();
+
+            // Assert
+            setTimeout(() => {
+                expect(FontFaceObserver).toBeCalled();
```

Comment:
Should use `toHaveBeenCalled` instead here as you want to check the actual mock is being called.